src/Sentry.OpenTelemetry/SentrySpanProcessor.cs

```diff
@@ -105,10 +105,20 @@ public override void OnEnd(Activity data)
         if (attributes.TryGetTypedValue("http.url", out string? url) && (_options?.IsSentryRequest(url) ?? false))
         {
             _options?.DiagnosticLogger?.LogDebug($"Ignoring Activity {data.SpanId} for Sentry request.");
-            if (_map.TryRemove(data.SpanId, out var removed) && (removed is SpanTracer spanTracerToRemove))
+
+            if (_map.TryRemove(data.SpanId, out var removed))
             {
-                spanTracerToRemove.IsSentryRequest = true;
+                if (removed is SpanTracer spanTracerToRemove)
+                {
+                    spanTracerToRemove.IsSentryRequest = true;
+                }
+
+                if (removed is TransactionTracer transactionTracer)
+                {
+                    transactionTracer.IsSentryRequest = true;
+                }
```

Comment:
You could do a switch here... I'm not sure if it's any more/less readable (up to you). 
```csharp
                switch (removed)
                {
                    case SpanTracer spanTracerToRemove:
                        spanTracerToRemove.IsSentryRequest = true;
                        break;
                    case TransactionTracer transactionTracer:
                        transactionTracer.IsSentryRequest = true;
                        break;
                }
```

Reply:
`switch` works as well. I'd go for it if it would be more than 2 cases. For 2 options, I'll keep it as is.